frontend/simple/views/Contributions.vue

```diff
@@ -1,69 +1,78 @@
 <template>
   <main>
-    <div class="section is-hero">
-      <i18n tag="h1" class="title is-1 is-marginless">
-        Contributions
-      </i18n>
-      <i18n tag="p">See everyone’s contributions to the group and manage yours.</i18n>
+    <div class="section is-hero level">
+      <div>
+        <i18n tag="h1" class="title is-1 is-marginless">
+          Contributions
+        </i18n>
+        <i18n tag="p">See everyone’s contributions to the group and manage yours.</i18n>
+      </div>
+      <div>
+        <groups-min-income :group="currentGroupState" />
+      </div>
     </div>
 
-    <div class="section columns is-desktop is-multiline c-grid">
-      <section class="column">
+    <section class="section columns is-desktop is-multiline c-grid">
+      <div class="column">
         <i18n tag="h2" class="title is-3">Receiving</i18n>
 
         <ul class="c-ul">
-          <contribution
-            v-for="contribution, index in receiving"
-            :variant="isMonetary(contribution.type) ? 'editable' : undefined"
-            :isMonetary="isMonetary(contribution.type)"
-            @interaction="showReceivingSettings"
-          >
-            <span v-html="textReceivingContribution(contribution, index)"></span>
-            <template v-if="hasWhoElse(contribution.who)">
-              <i18n>and</i18n>
-              <tooltip>
-                <button class="gi-is-unstyled gi-is-link-inherit">{{contribution.who.length - 1}}<i18n>others</i18n></button>
-                <template slot="tooltip">
-                  <p v-for="name, index in contribution.who" v-if="index > 0">{{name}}</p>
-                </template>
-              </tooltip>
-            </template>
+          <contribution v-for="contribution in receiving.nonMonetary">
+            <span v-html="textReceivingNonMonetary(contribution)"></span>
+            <TextWho :who="contribution.who"></TextWho>
```

Comment:
@sandrina-p can we use `<text-who :who="contribution.who"></text-who>` for consistency?

Reply:
oh, I didn't even noticed it! Sure, fixed!